Automattic/newspack-newsletters
# Posts Inserter block - handle before insertion state

### All Submissions:






### Changes proposed in this Pull Request:



If the posts are not inserted, they will be rendered in email as they are, without changes. 
The posts' blocks, as they _would_ be inserted, are stored in `innerBlocksToInsert` attribute.

Closes #114

### How to test the changes in this Pull Request:

1. Create a newsletter, add "Posts Inserter" block. Do **not** click "Insert posts"
2. Preview or send email - observe the posts are displayed, as if they were inserted
3. Change the settings of the block; repeat step 2 – changes should be reflected
4. Click "Insert Posts" button – observe the "Posts Inserter" blocks being replaced with the core blocks
5. Edit the blocks, repeat step 2 – changes should be reflected

### Other information:
